### PR TITLE
adds fake customer data in db.json

### PR DESCRIPTION
**Pull request by ivwu**

### CODE REVIEW

File: src/utils/http-common.js [hunk 1]
```diff
@@ -2,10 +2,12 @@ import axios from "axios";
 
 const { NODE_ENV } = process.env;
 
-const baseURL =
-  NODE_ENV === "development"
-    ? "http://localhost:3004/api"
-    : "https://amexcsrapi.azurewebsites.net/api";
+// const baseURL =
```
Comment (kchia) [inline src/utils/http-common.js:5]: @ivwu Make sure this is re-enabled before merging

Comment (kchia) [reply src/utils/http-common.js:5]:
```
const baseURL =  NODE_ENV === "development"
    ? "http://localhost:3004/api"
    : "https://amexcsrapi.azurewebsites.net/api";
```

**Review by kchia — CHANGES_REQUESTED** [review]
see comment

File: src/utils/http-common.js (hunk 1 above)
Comment (ivwu) [reply src/utils/http-common.js:5]: should i re-enable it right now or when the backend is done completing the model for customers

Comment (kchia) [reply src/utils/http-common.js:5]: You can re-enable it now. We do want to still keep the production site connected to the production backend at all times, even if some backend features are incomplete.

**Pull request closed without merging**